install/source_install.markdown

```diff
@@ -0,0 +1,94 @@
+---
+author: davetcoleman
+comments: false
+date: 2016-8-4 20:43:44+00:00
+layout: page
+slug: source_install
+title: source_install
+---
+
+# Source Installation Instructions
+
+For developers.
+
+## Prerequisites
+
+Install ROS [Indigo](http://www.ros.org/wiki/indigo/Installation/Ubuntu), [Jade](http://www.ros.org/wiki/jade/Installation/Ubuntu), or [Kinetic](http://www.ros.org/wiki/kinetic/Installation/Ubuntu). Please make sure you have followed all steps, including calls to ``rosdep``.
+
+Source installation requires [wstool](http://wiki.ros.org/wstool), and [catkin_tools](https://catkin-tools.readthedocs.io/en/latest/):
+
+    sudo apt-get install python-wstool python-catkin-tools
+
+Optionally create a new workspace, you can name it whatever:
+
+    mkdir -p ~/ws_moveit/src
+    cd ~/ws_moveit/src
+
+Follow the instrucitons below for either Indigo, Jade, or Kinetic:
+
+## Indigo
+
+Pull down required repositories and build from within the ``/src`` directory of your catkin workspace:
+
+    wstool init .
+    wstool merge https://raw.githubusercontent.com/ros-planning/moveit/indigo-devel/moveit.rosinstall
+    wstool update
+    rosdep install --from-paths . --ignore-src --rosdistro indigo
+    cd ..
+    catkin config --extend /opt/ros/indigo --install --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Comment:
I really do not see how this is any more simple than `. /opt/ros/indigo/setup/bash; catkin_make install -DCMAKE_BUILD_TYPE=Release` .
But as all the tests use catkin-tools now, I'm fine with keeping it that way.